### DISCUSSION TITLE
Erigon Archive Node across multiple disks?

### DISCUSSION THREAD

**Discussion by bcmeireles** [erigontech/erigon#13462] [Q&A]
Would it also be possible to have the current syncing be done in a SDD while the others disks are HDD that only store past blocks?
And prune on the SSD to never fill it up while sending the blocks that would be pruned to the HDDs

**Comment by AskAlexSharov** [comment]
yes, but only for blocks now (not state history, receipts) https://github.com/ledgerwatch/erigon?tab=readme-ov-file#datadir-structure

**Comment by AskAlexSharov** [comment]
done https://github.com/erigontech/erigon?tab=readme-ov-file#datadir-structure
